roles/ovirt-datacenter-cleanup/defaults/main.yml

```diff
@@ -1 +1,4 @@
 format_storages: false
+hosted_engine: false
```

Comment:
Is `false` reasonable default?

Reply:
Not sure is hosted_engine installations are more frequent. I sure can change this one, no problem.